### DISCUSSION TITLE
タグ検索で表示されない記事がある

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#890] [要望・アイデア・不具合]
LaTeX のタグページ (https://qiita.com/tags/latex) では以下の記事が表示されます。

wrapstuff で図表に文章を回り込みさせたい
(https://qiita.com/Yarakashi_Kikohshi/items/4922deb4038160bf796d)

しかし、新着順のタグ検索で `tag:latex` (https://qiita.com/search?q=tag%3Alatex&sort=created) と検索したときにはこの記事が表示されません。

タグ検索から記事を探すこともあるので、表示されない記事があると困ってしまいます 😖

**Comment by acronhub** [comment]
@Yarakashi-Kikohshi 

原因はわかっていないのですが、特定の状況で検索システムに反映がされない場合があります。
対応策として、定期的に検索システムに反映するようにしているので、現在は解消されておりました。
今後同症状が現れる場合は、また教えていただけると幸いです 🙇

**Reply by yarakos95** [reply]
@acronhub 様

現在、問題が解消されていることを確認しました。

同じような問題を発見できた場合は報告させていただきます。
加えて、原因が究明され問題が解消されることを願っています 🙏

-----

答えられる範囲で構わないのですが、1 つだけ質問させてください。

> 定期的に検索システムに反映するようにしている

定期的というのは週一程度ということでしょうか。検索で記事が引っかからない可能性がある期間が最大でどの程度あるのか知っておきたいです。

**Reply by acronhub** [reply]
@Yarakashi-Kikohshi

はい。週に一度の間隔で行っております。
ご不便をおかけして申し訳ありませんが、ご了承いただけますと幸いです🙇

**Reply by yarakos95** [reply]
@acronhub 様

ご回答ありがとうございます。
週一であれば、タグページで探して取りこぼしに気付ける量なので良かったです 😊
